src/pre.l

```diff
@@ -3439,7 +3439,9 @@ static void readIncludeFile(yyscan_t yyscanner,const QCString &inc)
     std::unique_ptr<FileState> fs;
     bool alreadyProcessed = FALSE;
     //printf("calling findFile(%s)\n",qPrint(incFileName));
-    if ((fs=findFile(yyscanner,incFileName,localInclude,alreadyProcessed))) // see if the include file can be found
+    fs=findFile(yyscanner,absIncFileName,localInclude,alreadyProcessed); // see if the absolute include file can be found
+    if (!fs && !alreadyProcessed) fs=findFile(yyscanner,incFileName,localInclude,alreadyProcessed); // see if the include file can be found
```

Comment:
Why was this not resolved by `determineAbsoluteIncludeName()`? Don't think it is correct to look relative from where doxygen is run. If this is wanted also `INCLUDE_PATH=.` would also do the trick, I'd expect.

Reply:
The Adding of `INCLUDE_PATH=.` does not solve the problem.
The `determineAbsoluteIncludeName()` crates an absolute path but the result (`absIncFileName`) was not given to the `findFile` routine, here the `incFileName` was given.
Probably just changing `incFileName` into `absIncFileName` would have solved the issue as well (so in the patch `if (!fs && !alreadyProcessed) fs=findFile(yyscanner,incFileName,localInclude,alreadyProcessed);` could be removed), but I wanted to play save.

Reply:
I don't like the extra `findFile()` call. After some time nobody understands anymore why it is there. So I've taken the risk of removing it.

Reply:
I think that this should be OK, tested 7z and looks OK.